paimon-core/src/main/java/org/apache/paimon/mergetree/compact/PartialUpdateMergeFunction.java

```diff
@@ -441,20 +439,6 @@ default Long generateNullable(InternalRow row, int i) {
 
         private static class SequenceGeneratorVisitor extends DataTypeDefaultVisitor<Generator> {
 
-            @Override
```

Comment:
Why remove here?

Reply:
The doc doesn't mention support for STRING, so I removed it. Otherwise, using STRING might throw errors.
> For fields.<field-name>.sequence-group, valid comparative data types include: DECIMAL, TINYINT, SMALLINT, INTEGER, BIGINT, FLOAT, DOUBLE, DATE, TIME, TIMESTAMP, and TIMESTAMP_LTZ.

Reply:
What we need to do is supporting string using `UserDefinedSeqComparator` way.

If you are interested in this, you can do it in a separate PR, please let this PR clean.

Reply:
ok, the code has been restored. will submit a new pr.